scripts/v120_polyline.py

```diff
@@ -19,6 +19,6 @@
     pts.append([random.uniform(0, 10), random.uniform(0, 10), random.uniform(0, 10)])
 
 polyline = Polyline(pts)
-viewer.add(polyline, show_points=True, color_line=(1, 0, 0), line_width=5)
+viewer.add(polyline, show_points=True, color_lines=(1, 0, 0), width_lines=5)
```

Comment:
we need to agree on better conventions for this. otherwise it is a bit of a guessing game as to what keywords arguments have to be used. in general, i noticed, for example, that python never uses underscores for this kind of things. it is also not the case for mpl...

Reply:
Maybe something like this? 

Point:
- pc = pt(s) color
- pw = pt(s) width

Line:
- lc = line(s) color
- lw = line(s) width

Face:

- fc = face(s) color
- fo = face(s) opacity